config.yaml

```
@@ -19,6 +19,10 @@ options:
       Cloning depth, to truncate commit history to the specified number of commits.
       Zero means no truncating.
     default: 1
+  git_ssh_key:
+    type: string
+    description: >
+      An optional SSH key to use to clone the repository.
```

```suggestion
      An optional SSH private key to use when cloning the repository.
```

The assumption is that it should be a privkey, but we should probably make that as clear as possible.